src/components/NotificationListContent.tsx

```diff
@@ -1,75 +1,302 @@
 import React, { useState } from 'react';
+import { View, Image, Text, StyleSheet, Platform } from 'react-native';
 import {
-    View,
-    Image,
-    Text,
-    StyleSheet,
-} from 'react-native';
-import { widthPercentageToDP as wp, heightPercentageToDP as hp } from 'react-native-responsive-screen';
-import Colors from "../common/Colors";
-import Fonts from "../common/Fonts";
-import CommonStyles from "../common/Styles";
-import { RFValue } from "react-native-responsive-fontsize";
-import ContactList from "../components/ContactList";
```

Comment:
@cakesoft-shivani 
CommonStyles, ContactList, BottomInfoBox, Ionicons, useState are declared but not used.
Let's remove them please 😊